docs/contributing/code-style.md

```diff
@@ -145,15 +145,15 @@ string, use the `id` function, as it will simplify future code changes.
 In most contexts in JavaScript where a string is needed, you can pass a
 number without any explicit conversion.
 
-### JavaScript var
+### JavaScript and Typescript `var`
 
-Always declare JavaScript variables using `var`.  JavaScript has
+Always declare JavaScript and TypeScript variables using `var`.  These languages have
```

Comment:
Actually in ES6 and Typescript we prefer to use `const` and `let` since they are block scoped. If I remember correctly, the TS compiler would warn you if a variable can be declared with a stricter declaration.